doc/buckets.md

```diff
@@ -1,122 +1,164 @@
-# Cloud Bucket configuration
+# Custom buckets
 
-Supported Cloud Providers:
 
-* **gcp**: [Google Storage](#google-storage) _default_
-* **azure-blob**: [Azure Blob Storage](#azure-blob-storage)
-* **s3**: [AWS S3](#aws-s3)
+For every CARTO Self Hosted installation, we create GCS buckets in our side as part of the required infrastructure for importing data, map thumbnails and other internal data. 
 
-Three different buckets are needed for CARTO Self Hosted: two for imports and one to store maps thumbnails.
+You can create and use your own storage buckets in any of the following supported storage providers:
 
-Map thumbnails can be configured in two ways. Public (thumbnails are public) or private and signed URLs are generated for them. In order to control it you need to change the variable: `WORKSPACE_THUMBNAILS_PUBLIC='true'`.
+- Google Cloud Storage
+- AWS S3
+- Azure Storage
 
-## Google Storage
+> :warning: You can only set one provider at a time.
 
-You need 3 buckets and a Service Account that can read/write to them.
+<!--
```

Comment:
Instead of the code, just add a link to the proper `.tf` file in `carto-selfhosted/terraform/examples`